settings.gradle

```diff
@@ -183,25 +183,29 @@ dependencyResolutionManagement {
             version( "h2gis", h2gisVersion )
             version( "hsqldb", hsqldbVersion )
             version( "derby", derbyVersion )
-            version( "pgsql", "42.5.0" )
-            version( "mysql", "8.0.27" )
-            version( "oracle", "21.8.0.0" )
+            version( "pgsql", "42.6.0" )
+            version( "mysql", "8.0.33" )
+            version( "oracle", "23.2.0.0" )
+            version( "mariadb", "2.7.9" )
```

Comment:
Any specific reason to not have opted for 3.1.4 (latest of mariadb) ?
 - https://mariadb.com/kb/en/about-mariadb-connector-j/

Reply:
Yes because of https://hibernate.atlassian.net/browse/HHH-15889?filter=21172